test/apps/iodemo/io_demo.cc

```diff
@@ -1710,14 +1710,14 @@ class DemoClient : public P2pDemoCommon {
             op_info[op].num_iters++;
 
             if (is_control_iter(total_iter) && (total_iter > total_prev_iter)) {
-                /* Print performance every 1 second */
```

Comment:
why removed?

Reply:
The interval is ```opts().print_interval``` https://github.com/openucx/ucx/blob/master/test/apps/iodemo/io_demo.cc#L1715
This could be changed by user instead of fixed time interval .i.e ```1 second```